### PR TITLE
fix: pass labelStyleProps down to FormLabel

### PR DESCRIPTION
**Pull request by Joaoviana**
Related to this issue: https://github.com/koolamusic/chakra-ui-autocomplete/issues/24

@koolamusic updated snapshots, anything else I'm missing? 😄

### CODE REVIEW

**Comment by koolamusic** [comment]
This looks good, thanks for sending this. I will review and respond shortly.

File: src/index.tsx [hunk 1]
```diff
@@ -191,7 +191,7 @@ export const CUIAutoComplete = <T extends Item>(
 
   return (
     <Stack>
-      <FormLabel {...getLabelProps({})}>{label}</FormLabel>
+      <FormLabel {...labelStyleProps} {...getLabelProps({})}>{label}</FormLabel>
```
Comment (koolamusic) [inline src/index.tsx:194]: Do you think we can do something like : 
`<FormLabel {...labelStyleProps, ...getLabelProps({})}>{label}</FormLabel>`

**Review by koolamusic — COMMENTED** [review]
@Joaviana please let me know your thoughts on this.

File: src/index.tsx (hunk 1 above)
Comment (Joaoviana) [reply src/index.tsx:194]: I think so! thanks for the suggestion 😄

**Comment by Joaoviana** [comment]
done @koolamusic, LMK if you prefer this approach 👍

**Comment by koolamusic** [comment]
Looks good to me. Thanks for making this happen

**Pull request merged**